<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisPropDialog</class>
 <widget class="QDialog" name="AnalysisPropDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>194</width>
    <height>209</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>168</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>157</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="covCheckBox">
      <property name="text">
       <string>Coverage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="SOLinkCheckBox">
      <property name="text">
       <string>Space Object Link</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="GOLinkCheckBox">
      <property name="text">
       <string>Ground Object Link</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>8</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select calculation</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>157</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
